templates/base/Dockerfile

```diff
@@ -3,10 +3,28 @@ FROM --platform=${TARGETPLATFORM} ubuntu:noble AS ubuntu-noble
 COPY resources/etc/created-date /etc/created-date
 
 RUN apt-get update && \
-	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes adduser bash ca-certificates curl jq less libnss3 telnet tini tree unzip && \
+	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes adduser bash build-essential ca-certificates curl jq less libnghttp2-dev libnss3 libssl-dev nghttp2 telnet tini tree unzip && \
```

Comment:
@Wykenz, can you please remove the additional packages as you did in https://github.com/natocesarrego/liferay-docker/pull/31/commits/40db715f245cc6a32948f86a9676c2df711edc8a?